Meta/Lagom/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required (VERSION 3.0)
 project (Lagom)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unknown-warning-option -O2 -Wall -Wextra -Werror -std=c++2a -fPIC -g -Wno-deprecated-copy")
```

Comment:
this is needed if your clang is new enough (i think 10+), which it has to be for the concept stuff

Reply:
Huh, interesting. I never ran into trouble with my clang version 9.0.1-13 o.O

Alright, I'll upgrade my system (somehow?) and remove the commit.

Reply:
Removed.